tests/test_currency_network_basics.py

```diff
@@ -30,6 +30,17 @@ def currency_network_contract_with_trustlines(currency_network_contract, account
     return contract
 
 
+@pytest.fixture()
+def currency_network_contract_custom_interest(chain):
+    CurrencyNetworkFactory = chain.provider.get_contract_factory('CurrencyNetwork')
+    deploy_txn_hash = CurrencyNetworkFactory.deploy(args=[])
+    contract_address = chain.wait.for_contract_address(deploy_txn_hash)
+    contract = CurrencyNetworkFactory(address=contract_address)
+    contract.transact().init('TestCoin', 'T', 6, 0, 0, True, False)
+    # init(name, symbol, decimal, feeDivisor, defaultInterest, customInterests)
```

Comment:
What is this? Comments do not help if they are wrong, so please just name the parameters.